### PR TITLE
Samsung Tizen

### PR DESCRIPTION
**Pull request by OmidID**
## What does the pull request do?
Implementation of the Tizen platform #6110 

## What is the current behavior?
New feature

## What is the updated/expected behavior with this PR?
1. Setup Tizen SDK https://github.com/Samsung/Tizen.NET
2. Run the ControlCatalog.Tizen

## Checklist

- [X] Added unit tests (if possible)?
- [x] Added XML documentation to any related classes?
- [X] Consider submitting a PR to https://github.com/AvaloniaUI/Documentation with user documentation
- [x] Pipeline update to build the Tizen platform
- [X] Fixing permission crash issue (https://github.com/Samsung/Tizen.NET/issues/250)
- [x]  https://github.com/Samsung/Tizen.NET/issues/249

### CODE REVIEW

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0037826-beta`. (feed url: https://nuget-feed-all.avaloniaui.net/v3/index.json) [PRBUILDID]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0037993-beta`. (feed url: https://nuget-feed-all.avaloniaui.net/v3/index.json) [PRBUILDID]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0038289-beta`. (feed url: https://nuget-feed-all.avaloniaui.net/v3/index.json) [PRBUILDID]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0038313-beta`. (feed url: https://nuget-feed-all.avaloniaui.net/v3/index.json) [PRBUILDID]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0038397-beta`. (feed url: https://nuget-feed-all.avaloniaui.net/v3/index.json) [PRBUILDID]

File: src/Tizen/Avalonia.Tizen/TizenStorageProvider.cs
Comment (maxkatz6) [inline src/Tizen/Avalonia.Tizen/TizenStorageProvider.cs:14]: it says "false", but OpenFolderPickerAsync actually is implemented

File: src/Tizen/Avalonia.Tizen/TizenThreadingInterface.cs
Comment (maxkatz6) [inline src/Tizen/Avalonia.Tizen/TizenThreadingInterface.cs:25]: I feel like this change would make sense it this place of code:
```suggestion
            MainloopContext.Post(static arg => ((Action)arg!).Invoke(), action);
```

Comment (maxkatz6) [reply src/Tizen/Avalonia.Tizen/TizenThreadingInterface.cs:25]: To avoid allocations on this branch

File: src/Tizen/Avalonia.Tizen/Platform/ScalingInfo.cs
Comment (maxkatz6) [inline src/Tizen/Avalonia.Tizen/Platform/ScalingInfo.cs:5]: I think it's not supposed to be public. Other classes should be also double-checked if visibility is expected.

File: src/Tizen/Avalonia.Tizen/NuiViewControlHandle.cs
Comment (maxkatz6) [inline src/Tizen/Avalonia.Tizen/NuiViewControlHandle.cs:11]: I think the descriptor should be a bit more specific about platform. As "View" doesn't tell anything about tizen or Nui.

File: src/Tizen/Avalonia.Tizen/Platform/ScalingInfo.cs
Comment (OmidID) [reply src/Tizen/Avalonia.Tizen/Platform/ScalingInfo.cs]: File removed.

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0038408-beta`. (feed url: https://nuget-feed-all.avaloniaui.net/v3/index.json) [PRBUILDID]

**Comment by OmidID** [comment]
Hello @maxkatz6 .
Any update for this PR?

**Comment by maxkatz6** [comment]
@OmidID it looks good to me. But was waiting for the review from somebody else, as it's a big feature. Will ping the team. Thanks!

**Review by maxkatz6 — APPROVED** [review]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0039774-beta`. (feed url: https://nuget-feed-all.avaloniaui.net/v3/index.json) [PRBUILDID]

**Pull request merged**
